comictalker/__init__.py

```diff
@@ -30,7 +32,15 @@ def get_talkers(version: str, cache: pathlib.Path) -> dict[str, ComicTalker]:
             if obj.id != talker.name:
                 logger.error("Talker ID must be the same as the entry point name")
                 continue
-            talkers[talker.name] = obj
+            try:
+                if parse(version) >= parse(obj.comictagger_min_ver):
+                    talkers[talker.name] = obj
+                else:
+                    logger.error(
+                        f"CT minimum required version not met for talker: {talker.name} with version: {obj.comictagger_min_ver}"
```

Comment:
CT should be spelled out. To me this reads that the version is the talker version and not ComicTagger's version